frontend/src/components/ShootWorkers/MachineType.vue

```diff
@@ -121,14 +166,40 @@ export default {
     },
     hint () {
       return this.notInList ? 'This machine type may not be supported by your worker' : ''
+    },
+    cpuItems () {
+      const cpuItems = uniq(map(this.machineTypes, 'cpu'))
+      return ['all', ...cpuItems.sort((a, b) => (a - b))]
```

Comment:
what if cpu is in milli format? e.g. 1000m=1(, 100m=0.1) https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-cpu

Reply:
I this the case for machines? I thought this only applies to pods?